piff/star_stats.py

```diff
@@ -115,7 +115,8 @@ def compute(self, psf, stars, logger=None):
                                      axis=0)
             ave_model_image = np.mean([s.image.array/np.max(s.image.array)
                                         for s in calculated_models
-                                        if s is not None and not s.is_flagged],
+                                        if s is not None and not s.is_flagged
+                                            and np.max(s.image.array) != 0.],
```

Comment:
Is it a problem that max(image) == 0 models have survived this far?  Stats feels kinda late in the game to find this.

Reply:
I think it's super rare.  But there was one case where a model centroid moved by more than the size of the stamp, so the whole model was outside the image.  I don't think we currently have any limit of the amount of centroid shift that is allowed, but it probably makes sense to add something.  This only happened when the model was not really working other reasons anyway, but I didn't want the division by zero in the average.